src/assets/pages/blog.json

```
@@ -1,5 +1,23 @@
 {
   "blog": [
+    {
+      "id": 32,
+      "title": "Salvador: as raízes de uma 'cidade de papel'",
+      "category":  "Querido Diário nas Universidades",
```

```suggestion
      "category":  "#AcheiNoDiárioOficial",
```

Agora que o #150 foi aceito, me fez lembrar de vir verificar aqui qual a categoria. Em "Querido Diário nas Universidades" só tem informações operacionais do programa, os achados (reportagens e matérias) feitas a partir de pesquisa nos dados do QD ficam em "[#AcheiNoDiárioOficial](https://queridodiario.ok.org.br/blog/%23AcheiNoDi%C3%A1rioOficial)"